cegs_portal/search/urls.py

```diff
@@ -67,4 +67,5 @@
     ),
     re_path(r"v1/regeffect/(?P<re_id>DCPREO[A-F0-9]{8})$", views.v1.RegEffectView.as_view()),
     path("v1/feature_counts", views.v1.FeatureCountView.as_view()),
+    re_path(r'regeffect/nontarget/download_tsv/(?P<feature_id>DCP[A-Z]{1,4}[A-F0-9]{8})/$', views.v1.DownloadTSVView.as_view(), name='download_proximal_regulatory_observations_table_data_tsv'),
```

Comment:
This line should be moved up under the other `regeffect/nontarget` URL. 

You may have noticed that most of the URLs are kind of listed twice -- once with a `v1` prefix and once without. This is so we can give people stable API endpoints -- the `v1/` endpoints should always kind of be the same -- the data format shouldn't change too much once we go to production. If someone is writing some scripts to programmatically use the portal they should be using the `v1` endpoints. The other endpoints are more mutable and don't need to be stable. 

If we ever have a "v2" we would create a new set of `v2` endpoints and make the non-version-prefixed urls use those.